boards/lilygo/ttgo_lora32/ttgo_lora32-pinctrl.dtsi

```diff
@@ -21,6 +21,18 @@
 		};
 	};
 
+	spim2_default: spim2_default {
```

Comment:
this is not related to SDHC, right?

Reply:
Hello Raffael,
no, thankse.  I removed that.